hugo.toml

```diff
@@ -8,3 +8,6 @@
     baseName = 'llms'
     mediaType = 'text/plain'
     isPlainText = true
+  [outputFormats.markdown]
```

Comment:
is it necessary here as hugo has `markdown` output format builtin?

https://gohugo.io/configuration/output-formats/